templates/sounds/latest_sounds.html

```diff
@@ -0,0 +1,20 @@
+{% load cache %}
+{% load display_sound %}
+
+<!-- recent additions -->
+{% cache 600 latest_sounds %}
+<div id="recent_additions" class="content_box">
+    <h3>Recent Additions</h3>
+    {% for sound in latest_sounds %}
+        {% display_sound sound.id %}
```

Comment:
display_sound will make a number of additional queries per sound to get all of the relevant information (username, tags, license). We could modify the latest sounds query to be like get_raw_sound and collect this information as well and then use display_raw_sound. However, we're also caching this for 10 minutes, so maybe it's worth leaving it as-is in exchange for making it a bit easier to understand (I'm sure this comment will come back and get us in the future)

Reply:
we decided that this is OK for now, especially with the cache.